site/docs/v1/tech/apis/system.adoc

```diff
@@ -442,8 +444,10 @@ The version of the running FusionAuth instance.
 
 This page contains the API that is used for retrieving FusionAuth application metrics to be used with Prometheus. Please refer to the link:/docs/v1/tech/tutorials/prometheus#[Prometheus setup] guide to understand how to set up Prometheus with the FusionAuth metrics endpoint.
 
+By default, this API requires authentication. If you prefer to allow unauthenticated access to this endpoint from local scrapers, you may set `fusionauth-app.local-metrics.enabled=true` as link:/docs/v1/tech/reference/configuration#[documented here].
+
 [.api-authentication]
-link:/docs/v1/tech/apis/authentication#basic-authentication-using-an-api-key[icon:shield-alt[type=fas]] link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
+link:/docs/v1/tech/apis/authentication#no-authentication-required[icon:unlock[type=fas]] link:/docs/v1/tech/apis/authentication#basic-authentication-using-an-api-key[icon:shield-alt[type=fas]] link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
```

Comment:
Shouldn't overload the unlock icon. I'd pick something ip based as the icon and add it as a different kind of authentication here.

Reply:
I changed it to use the `server` icon, which seemed to fit more than anything else. I added it as part of https://github.com/FusionAuth/fusionauth-style/pull/29.